context-monitoring/src/main/java/de/atb/context/monitoring/index/Indexer.java

```diff
@@ -291,7 +291,7 @@ protected final void createIndexWriter() {
         try {
             FSDirectory dir = FSDirectory.open(tmpDir.toPath());
             this.writer = new IndexWriter(dir, indexWriterConfig);
-
+            this.writer.commit();
             // Index does not exist => create!
```

Comment:
This is related to the catch block below? Shouldn't it be the first line inside the catch block?

Also: lines 304 to 312 are duplicating lines 315 to 323. This should be refactored.